src/hotspot/share/runtime/threads.cpp

```diff
@@ -552,6 +554,15 @@ jint Threads::create_vm(JavaVMInitArgs* args, bool* canTryAgain) {
   // crash Linux VM, see notes in os_linux.cpp.
   main_thread->stack_overflow_state()->create_stack_guard_pages();
 
+  if (UseSystemMemoryBarrier) {
+    if (!SystemMemoryBarrier::initialize()) {
+      // UseSystemMemoryBarrier = false;
```

Comment:
Delete the commented out line? Or was this done for testing? Or???

Reply:
Yes, thanks, fixed.